docs/src/1_Mechanics/04_Powers.yaml

```
@@ -825,6 +827,87 @@ Intensify:
     Description: An explosive attack might mean running out of gas.
     Mechanic: Discard no cards. Your next attack is treated as an Epic Event. Discard as many cards as you choose until a Major Success. Standard Successes are treated as Misses
     Type: Minor
-    XP: 0 
+    XP: 0
     Prereq:
         Level: 2
+
+Favored Terrain:
+    Type: Passive
+    Category:
+        - Movement
+        - Speed
+    XP: 1
+    Description: Home feels more familiar, granting a speed boost.
+    Mechanic: Work with your GM to decide a favored terrain (e.g., mountains). Gain +2 speed on this terrain. 
+
+Terrain Blink:
+    Type: Minor
+    Category:
+        - Movement
+        - Speed
+    XP: 3
+    Description: So at home that you can get anywhere.
+    Mechanic: Work with your GM to decide a favored terrain (e.g., mountains). When navigating this contiguous terrain, you can use half your movement to immediately move to a point you can see within 10 spaces.
+
+Condition Resilience: 
+    Type: Passive
+    Category: Resilience 
+    XP: 1
+    Prereq:
+        Skill: VIT > 1
+    Description: Through training or sheer will, you brace your body for trials ahead.
+    Mechanic: "Select on of the following conditions when taking this power: Blinded, Deafened, Burned, Poisoned, Frozen, or Suffocating. If you make a check to avoid being subjected to this effect, you do so at Upper Hand."
+
+Condition Immunity: 
+    Type: Passive
+    Category: Resilience 
+    XP: 3
+    Prereq:
+        Skill: VIT > 1
+    Description: Through training or sheer will, your body is impervious.
+    Mechanic: "Select on of the following conditions when taking this power: Blinded, Deafened, Burned, Poisoned, Frozen, or Suffocating. You are now immune to this condition."
+
+Adversary Major Action:
+    Type: Major
```

```suggestion
    Type: Adversary
```

Should Adversary be its own category/type since it is an action that is distinct in that it can be used *between* turns?